python/ray/util/placement_group.py

```diff
@@ -78,9 +78,13 @@ def bundle_count(self):
         return len(self.bundle_cache)
 
     def _get_none_zero_resource(self, bundle: List[Dict]):
+        # This number shouldn't be changed.
+        # When it is specified, node manager won't warn about infeasible
+        # tasks.
+        INFEASIBLE_TASK_SURPRESS_MAGIC_NUMBER = 0.0101
```

Comment:
SUPPRESS

Reply:
Oops.. Good to learn the right spell this time.. lol

Reply:
I made sure all typos were fixed (by searching surpress)